docs/tutorial.part-4.rst

```diff
@@ -0,0 +1,146 @@
+Part 4: Connecting to a `testrpc` Node
```

Comment:
Can we leave this out for now.  `testrpc` is poorly supported and I'm trying to replace it all-together with something backed by ethereum-tester.  If nothing else, you can convert this to use the javascript testrpc since it's well supported.